episodes/02-branching.md

```diff
@@ -164,6 +196,38 @@ All branches and commits exist within the collaborators fork, not the ***upstrea
 
 Collaborators can use their fork to test more complex changes. For example testing github actions within a dummy-PR.
 
+```mermaid
+---
+title: Forking + Feature Branch
+---
+flowchart TB
+  accDescr {A flowchart showing the use of forks
+    and the feature branch model.
+    Here developers work on their forks (GitHub copies, labelled origin) of the
+    main GitHub repository (upstream).
+    Developers work on feature branches in their fork,
+    and push these branches to the origin repository.
+    Pull requests are used to contribute changes from
+    the origin to the upstream repository.
+    Only one developers fork and local repository
+    are shown in this diagram.}
+  subgraph top["Remote Server (GitHub)"]
+    direction LR
+    r1[("MetOffice/git-training-demo Upstream Repository")]
+    r2[("username/git-training-demo Origin Repository")]
+
+    r2 -. Pull Request .-> r1
+  end
+  subgraph bottom["Computer"]
+    r3[("Local Repository")]
+  end
+    r1 -- fetch/pull --> r3
+    r3 -- push --> r2
+    r2 -- fetch/pull --> r3
```

Comment:
Slight inconsistency between the 2 diagrams (my fault). Either both should say `fetch/pull` or both should just say `pull`.